.travis.yml

```diff
@@ -2,7 +2,6 @@ sudo: false
 
 language: node_js
 node_js:
-  - "4"
```

Comment:
No, because tests won't pass since ESLint 5 does not support Node 4.

Reply:
Do you want the rest of this added to this PR or should I create another one?

Reply:
> > can you put that in a separate PR?
>
> No, because tests won't pass since ESLint 5 does not support Node 4.

dropping support for a Node version is a breaking change and should not be a side effect of another PR. as PRs will show up in the changelog there should be a dedicated PR just for that, which includes all the other changes that I listed above. I don't see why we can't have a PR for dropping Node 4 first, and then when that is merged, rebase this one on top of it.

Reply:
Ok, np. Will break this into 3 PRs:

1. Drop support for node 4
2. Update broccoli dependencies
3. Support ESLint 5

Reply:
awesome, thanks :)